xml/deploy_cephadm.xml

```diff
@@ -929,6 +929,847 @@ mon.ses-min1    ses-min1 running) 8m ago    12d 15.2.0.108 5bf12403d0bd a719e008
+osds_per_device: 1   # number of osd daemons per device
+encryption:           # 'True' or 'False' (defaults to 'False')
+</screen>
+    </sect4>
+    <sect4>
+     <title>Matching Disk Devices</title>
+     <para>
+      You can describe the specification using the following filters:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        By a disk model:
+       </para>
+<screen>
+model: <replaceable>DISK_MODEL_STRING</replaceable>
+</screen>
+      </listitem>
+      <listitem>
+       <para>
+        By a disk vendor:
+       </para>
+<screen>
+vendor: <replaceable>DISK_VENDOR_STRING</replaceable>
+</screen>
+       <tip>
+        <title>Lowercase Vendor String</title>
+        <para>
+         Always enter the <replaceable>DISK_VENDOR_STRING</replaceable> in
+         lower case.
+        </para>
+       </tip>
+      </listitem>
+      <listitem>
+       <para>
+        Whether a disk is rotational or not. SSDs and NVMe drives are not
+        rotational.
+       </para>
+<screen>
+rotational: 0
+</screen>
+      </listitem>
+      <listitem>
+       <para>
+        Deploy a node using <emphasis>all</emphasis> available drives for OSDs:
+       </para>
+<screen>
+data_devices:
+  all: true
+</screen>
+      </listitem>
+      <listitem>
+       <para>
+        Additionally, by limiting the number of matching disks:
+       </para>
+<screen>
+limit: 10
+</screen>
+      </listitem>
+     </itemizedlist>
+    </sect4>
+    <sect4>
+     <title>Filtering Devices by Size</title>
+     <para>
+      You can filter disk devices by their size&mdash;either by an exact size,
+      or a size range. The <option>size:</option> parameter accepts arguments
+      in the following form:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        '10G' - Includes disks of an exact size.
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        '10G:40G' - Includes disks whose size is within the range.
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        ':10G' - Includes disks less than or equal to 10&nbsp;GB in size.
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        '40G:' - Includes disks equal to or greater than 40&nbsp;GB in size.
+       </para>
+      </listitem>
+     </itemizedlist>
+     <example>
+      <title>Matching by Disk Size</title>
+<screen>
+service_type: osd
+service_id: example_drvgrp_name
+placement:
+ host_pattern: '*'
+data_devices:
+  size: '40TB:'
+db_devices:
+  size: ':2TB'
+</screen>
+     </example>
+     <note>
+      <title>Quotes Required</title>
+      <para>
+       When using the ':' delimiter, you need to enclose the size in quotes,
+       otherwise the ':' sign will be interpreted as a new configuration hash.
+      </para>
+     </note>
+     <tip>
+      <title>Unit Shortcuts</title>
+      <para>
+       Instead of (G)igabytes, you can specify the sizes in (M)egabytes or
+       (T)erabytes as well.
```

Comment:
```suggestion
       Instead of Gigabytes (G), you can specify the sizes in Megabytes (M) or
       Terabytes (T).
```

Reply:
Are these SI units or Mibi/Gibi/Tibi?  I suspect that should be clarified (unless it already is somewhere else).

I have removed the slightly "clever" formatting of the initial letter in parentheses, because it might not work well for people listening to the text, e.g. audiobook or visually-impaired users. Also, we don't need "instead" **and** "as well".

Reply:
it's the only units i know, 1024 bits multipled each time, as usually specified for storage space